README.md

```diff
@@ -3,12 +3,32 @@
 `http-proxy` is a ratelimited HTTP proxy in front of the Discord API, making use
 of [twilight].
 
-### Using it
+## Using it
 
-HTTP clients often support proxies, such as Ruby's [`Net::HTTP`]. Read into your
-HTTP client to see how to use it.
+This proxy is not a "real" HTTP `CONNECT` TCP proxy, instead it mocks to be the
+Discord API.
 
-`twilight_http` natively supports using `twilight_http_proxy`, so you can use it like
+In order to use it, change the base URL of all your routes from
```

Comment:
I think for this section, we should prefer to begin with explaining the simplest use case, which I understand to be `curl http://localhost:3000/users/@me`. We can then mention that the user can add `/v9/` or another version string to the URL, and be clear whether or not the `/api/` segment is required or optional.